Cargo.toml

```diff
@@ -7,13 +7,14 @@ edition = "2021"
 
 [dependencies]
 clap = { version = "4.3.0", features = ["derive"] }
-grep = "0.2.12"
+grep = { git = "https://github.com/helixbass/ripgrep", rev = "b83cc3c" }
 ignore = "0.4.20"
 libc = "0.2.144"
 libloading = "0.8.0"
 once_cell = "1.17.1"
 rayon = "1.7.0"
 regex = "1.8.2"
+termcolor = "1.2.0"
```

Comment:
When I had to write out the type of a `Searcher` (I think?) I needed to reference a type from `termcolor` which I guess is another one of the "`ripgrep`-derived" crates